spec.md

```diff
@@ -2450,6 +2450,26 @@ asClock(x)
 
 [@sec:primitive-operations] will describe the format and semantics of each primitive operation.
 
+## Primitive Property Operations {#sec:expressions:primitive-property-operations}
+
+All fundamental operations on property types are expressed as a FIRRTL primitive operation.
+In general, each operation takes some number of property type expressions as arguments.
+
+The general form of a primitive property operation is expressed as follows:
+
+```firrtl
+op(arg0, arg1, ..., argn)
+```
```

Comment:
Nit: Pandoc should want to reformat this to include a space:

```suggestion
``` firrtl
op(arg0, arg1, ..., argn)
```
```